### PR TITLE
Add NGINX Kubernetes Gateway to list of implementations

### PR DESCRIPTION
**Pull request by pleshakov**
**What type of PR is this?**
/kind documentation

**What this PR does / why we need it**:
The PR adds NGINX Kubernetes Gateway to the list of the implementations in the docs.

**Which issue(s) this PR fixes**:

None

**Does this PR introduce a user-facing change?**:

None

### CODE REVIEW

**Comment by youngnick** [comment]
/ok-to-test

Looks like you'll need to get the CLA squared away as well, @pleshakov.

Change lgtm, but I'll wait to push the button until the other stuff is fixed.

File: site-src/implementations.md [hunk 1]
```diff
@@ -130,6 +132,13 @@ Kong is actively working on an implementation of the Gateway API in the [Kong Ku
 [kic]:https://github.com/kong/kubernetes-ingress-controller
 [kong-gw-guide]:https://docs.konghq.com/kubernetes-ingress-controller/latest/guides/using-gateway-api/
 
+### NGINX Kubernetes Gateway
+
+[NGINX Kubernetes Gateway][nginx-kubernetes-gateway] is an open-source project that provides an implementation of the Gateway API using [NGINX][nginx] as the data plane. The goal of this project is to implement the core Gateway APIs -- Gateway, GatewayClass, HTTPRoute, TCPRoute, TLSRoute, and UDPRoute -- to configure an HTTP or TCP/UDP load balancer, reverse-proxy, or API gateway for applications running on Kubernetes. NGINX Kubernetes Gateway is currently under development and supports a subset of the Gateway API.
```
Comment (robscott) [inline site-src/implementations.md:137]: Tiny nit: Do you mind wrapping these lines at ~80 chars?

**Review by robscott — COMMENTED** [review]
Thanks @pleshakov! LGTM except for one tiny nit.

File: site-src/implementations.md (hunk 1 above)
Comment (robscott) [reply site-src/implementations.md:137]: Withdrawing this nit because it looks like we haven't required this for other implementations.

**Comment by robscott** [comment]
Thanks!

/lgtm
/approve

**Pull request merged**